File: CondicionalesCsharp/4.cs
using System;


namespace Condicionales
{
    internal class Cuatro
    {
        public Cuatro() {
            Console.WriteLine("Digita el primer número: ");
            int num1 = int.Parse(Console.ReadLine());

            Console.WriteLine("Digita el segundo número: ");
            int num2 = int.Parse(Console.ReadLine());

            if (num1 < num2)
            {
                Console.WriteLine(num1 + " Es menor que " + num2);
            }
            else if (num1 > num2)
            {
                Console.WriteLine(num1 + " es mayor que " + num2);
            }
            else
            {
                Console.WriteLine("Los números son iguales");
            }
        }
    }
}

File: dart/condicional/4.dart
import 'dart:io';

void main() {
  print("Número mayor o menor");
  
  print("Digite un número cualquiera.");
  
  int x = int.parse(stdin.readLineSync()!);
  
  print("Digite otro número cualquiera.");
  
  int y = int.parse(stdin.readLineSync()!);

  if(x > y){
      
      print("${x} Es mayor a ${y}.");
      
  }else if(x < y){
      
      print("${x} es menor que ${y}.");
      
  }else{
      
      print("Ambos números son iguales.");
      
  }
}
